src/client/java/teammates/client/scripts/sql/PatchCreatedAtAccountRequest.java

```diff
@@ -0,0 +1,246 @@
+        numberOfUpdatedEntities = new AtomicLong();
+
+        String connectionUrl = ClientProperties.SCRIPT_API_URL;
+        String username = ClientProperties.SCRIPT_API_NAME;
+        String password = ClientProperties.SCRIPT_API_PASSWORD;
+
+        HibernateUtil.buildSessionFactory(connectionUrl, username, password);
+    }
+
+    public static void main(String[] args) {
+        new PatchCreatedAtAccountRequest().doOperationRemotely();
+    }
+
+    /**
+     * Returns the log prefix.
+     */
+    protected String getLogPrefix() {
+        return String.format("Account Request Patch Migration:");
+    }
+
+    private boolean isPreview() {
+        return false;
+    }
+
+    /**
+     * Returns whether the account has been migrated.
+     */
+    protected boolean isMigrationNeeded(teammates.storage.entity.AccountRequest entity) {
+        return true;
+    }
+
+    /**
+     * Returns the filter query.
+     */
+    protected Query<teammates.storage.entity.AccountRequest> getFilterQuery() {
+        return ofy().load().type(teammates.storage.entity.AccountRequest.class);
+    }
+
+    private void doMigration(teammates.storage.entity.AccountRequest entity) {
+        try {
+            if (!isMigrationNeeded(entity)) {
+                return;
+            }
+            if (!isPreview()) {
+                migrateEntity(entity);
+            }
+        } catch (Exception e) {
+            logError("Problem patching account request " + entity);
+            logError(e.getMessage());
+        }
+    }
+
+    /**
+     * Migrates the entity. In this case, add entity to buffer.
+     */
+    protected void migrateEntity(teammates.storage.entity.AccountRequest oldEntity) {
+        HibernateUtil.beginTransaction();
+
+        CriteriaBuilder cb = HibernateUtil.getCriteriaBuilder();
+        CriteriaQuery<teammates.storage.sqlentity.AccountRequest> cr = cb.createQuery(teammates.storage.sqlentity.AccountRequest.class);
+        Root<teammates.storage.sqlentity.AccountRequest> root = cr.from(teammates.storage.sqlentity.AccountRequest.class);
+
+        cr.select(root).where(cb.equal(root.get("institute"), oldEntity.getInstitute()))
+            .where(cb.equal(root.get("email"), oldEntity.getEmail()))
+            .where(cb.equal(root.get("name"), oldEntity.getName()));
+
+        List<teammates.storage.sqlentity.AccountRequest> matchingAccounts = HibernateUtil.createQuery(cr).getResultList();
+        
+        if (matchingAccounts.size() > 1) {
+            throw new Error("More than one matching account request found");
+        } else if (matchingAccounts.size() == 0){
+            throw new Error("No matching account found");
+        }
+
+        // Get first items since there is guaranteed to be one
+        teammates.storage.sqlentity.AccountRequest newAccountReq = matchingAccounts.get(0);
+        newAccountReq.setCreatedAt(oldEntity.getCreatedAt());
+        HibernateUtil.commitTransaction();
```

Comment:
Since we're fetching an existing entity, changing the object attribute will save the changes in the DB